go/src/socialapi/workers/payment/customer.go

```diff
@@ -23,6 +22,8 @@ var (
 	ErrCustomerNotExists = errors.New("user is not created for subscription")
 	// ErrGroupAlreadyHasSub error when a group tries to create a sub and they try to create another
 	ErrGroupAlreadyHasSub = errors.New("group already has a subscription")
+	// ErrCustomerSourceNotExists holds the error when a customer does not have a card
+	ErrCustomerSourceNotExists = errors.New("does not have source")
```

Comment:
the errors message seems different from what the comment says - `errors.New("customer does not have a card"`?